src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -638,12 +648,69 @@ describe('<OfferBody />', () => {
       })
     })
   })
+
+  describe('same artist playlist with "wipSameArtistPlaylist" feature flag activated', () => {
+    beforeAll(() => {
+      mockSameArtistPlaylist = mockedAlgoliaOffersWithSameArtistResponse
+    })
+
+    beforeEach(() => {
+      useFeatureFlagSpy.mockReturnValueOnce(false).mockReturnValueOnce(true)
+    })
+
+    it('should display same artist list when offer has some', async () => {
+      renderOfferBody({
+        sameArtistPlaylist: mockSameArtistPlaylist,
+      })
+
+      await screen.findByText('Envoyer sur Instagram')
+
+      expect(screen.queryByTestId('sameArtistPlaylist')).toBeOnTheScreen()
```

Comment:
Pourquoi ne pas les fusionner ? Tu crées un dépendance aux réseaux sociaux qui n'est pas utile. Si un jour on n'en veut plus tu vas casser un test qui n'est pas concerné
```suggestion
      expect(await screen.getByTestId('sameArtistPlaylist')).toBeOnTheScreen()
```
Ca ne marche pas ça ?

Reply:
non ça ne marche pas